notebooks/01_data_preprocessing.ipynb

```diff
@@ -22,7 +22,8 @@
     "import scipy as sp\n",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    # Define the xarray_open_kwargs with a compatible engine, for example, 'scipy'
</pre><p>You might want to add h5netcdf why it is being used here. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NCAR/ERA5-dashboard-cookbook/pull/11/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>